flexible-classroom/reference/manage-agora-account.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Agora account management'
```

Comment:
Suggestion: Reorder this doc and move it to after "Platform support". 
Reason: There are many docs under the reference section. So I think it's better to show the reference docs in a kind of order principle, for example, show the reference doc related to Flexible Classroom first, and then show the universal reference doc related to Agora.

Reply:
Why not. Done.